tests/sentry/event_manager/test_hierarchical_hashes.py

```diff
@@ -202,54 +202,18 @@ def test_partial_move(default_project, fast_save):
 
 
 class EventManagerGroupingTest(TestCase):
-    def test_applies_secondary_grouping_hierarchical(self):
-        project = self.project
-        project.update_option("sentry:grouping_config", "legacy:2019-03-12")
-        project.update_option("sentry:secondary_grouping_expiry", 0)
```

Comment:
These 3 lines now live in the shared function `set_options`:
<img width="540" alt="image" src="https://github.com/getsentry/sentry/assets/44410/76059fc7-a953-4ce6-862f-14b42a8b4178">